src/compas/artists/surfaceartist.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+from compas.colors import Color
+from .artist import Artist
+
+
+class SurfaceArtist(Artist):
+    """Base class for artists for surfaces.
+
+    Parameters
+    ----------
+    surface: :class:`~compas.geometry.Surface`
+        The surface geometry.
+    color : tuple[float, float, float] | :class:`~compas.colors.Color`, optional
```

Comment:
the pipe is a new thing for `or`, I think we've been using the word `or` or the `Union[]` type so far

Reply:
i know but it is the preferred over `Union` in the latest versions of the python docs

Reply:
yes